scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -129,7 +127,7 @@ case class Args(
     @Description(
       "The Scala compiler version that was used to compile this project."
```

Comment:
Since we are extending the use of this flag for syntactic rules, I think we should relax a bit the constraint of "prior compilation". Also, the reference to `project` is a bit ambiguous, considering the other flags, I think `files` would be more consistent.
```suggestion
      "The Scala compiler version that the provided files are targeting, or the actual version that was used to compile them when a classpath is provided."
```